tests/autoclassify/utils.py

```diff
@@ -39,6 +40,8 @@ def create_failure_lines(job, failure_line_list,
         data.update(base_data)
         data.update(updates)
         failure_line = FailureLine(**data)
+        job_log = JobLog.objects.create(job=job, name='foo', url='bar{}'.format(i), status=1)
```

Comment:
Needed to add a ``log_line`` to connect it to a ``job`` because that's how we do it in ``tests.py``

Reply:
Do we need to fix `tests.py` instead?
I'm confused with the use of `foo` and `bar`.

Reply:
In this case, `foo` is just the name given to the log and `bar#` would be the URL.  But we won't actually be using those values.  They have to be unique, due to an index, but other than that, they're just a link to the job in question.